dotcom-rendering/src/components/Card/Card.stories.tsx

```diff
@@ -32,7 +32,7 @@ const basicCardProps: CardProps = {
 	imagePosition: 'top',
 	showAge: true,
 	isExternalLink: false,
-	videoSize: 'large enough to play: at least 480px',
```

Comment:
@mxdvl - this props seems to only be used to detemine if its playable, rather that its size so this PR opts for a different name. However, this is very personal-preferency so we'd be keen to know if there's other reasons for using this prop name with a hardcoded string as opposed to a boolean

Reply:
The reasoning behind an explicit string was that it would require one less click for the next maintainer. I originally went with a similar boolean, but quickly wondered wether it would be clear why the video was unplayable.

There could be a multitude of reasons:
- rights expired 
- a tools decision
- unavailable in region
- no duration
- not really a video
- …

It was a very non-standard decision, though, so happy for it to be reverted — especially if you find the boolean clearer and you're the owner of this repo! 😁

Reply:
Thanks for this Max!